SPECS/ocaml/ocaml.spec

```diff
@@ -194,51 +118,41 @@ make=make
 %configure \
     OC_CFLAGS="$CFLAGS" \
     OC_LDFLAGS="$LDFLAGS" \
+%if %{with_check}
+    --enable-ocamltest \
+%endif
     --libdir=%{_libdir}/ocaml \
     --host=`./build-aux/config.guess`
-$make world
-%if %{native_compiler}
-$make opt
-$make opt.opt
-%endif
-
-
-%check
-%ifarch %{test_arches}
-cd testsuite
-
-%ifarch %{test_arches_required}
-make -j1 all
-%else
-make -j1 all ||:
-%endif
-%endif
 
+%make_build world
+%make_build opt
+%make_build opt.opt
 
 %install
-make install DESTDIR=$RPM_BUILD_ROOT
-perl -pi -e "s|^$RPM_BUILD_ROOT||" $RPM_BUILD_ROOT%{_libdir}/ocaml/ld.conf
+%make_install
+perl -pi -e "s|^%{buildroot}||" %{buildroot}%{_libdir}/ocaml/ld.conf
 
-echo %{version} > $RPM_BUILD_ROOT%{_libdir}/ocaml/fedora-ocaml-release
+echo %{version} > %{buildroot}%{_libdir}/ocaml/fedora-ocaml-release
```

Comment:
Note: I would like to keep this file here, until I can figure out what pieces of tooling might rely on it.